packages/block-library/src/latest-posts/block.json

```diff
@@ -12,6 +12,12 @@
 		"categories": {
 			"type": "array"
 		},
+		"users": {
+			"type": "array"
```

Comment:
This makes the PR not work in my tests. Given that we only filter by one author and all we need is the ID which is a Number, we should have this set to "number"

Reply:
I agree. I've updated the type to number.